codecombat/codecombat
# [Prototype] Load an Adobe Animate movieclip directly

This proof of concept loads an Adobe Animate exported segmented sprite sheet directly onto the canvas.

See it work by loading any cinematic. Hero B should appear walking on the bottom right of the screen.

This gives us back the segmented sprite optimization through the Adobe Animate export process.
It also allows us to skip the client side rendering of sprite sheets and the error prone parsing of raw data!

## How to test

1. Run `npm run dev` with your local server running via `npm run nodemon`.
2. Create a new empty cinematic.
3. Add `shots` property to the cinematic.
4. Test the cinematic. There should be hero 2 running at the bottom right of the screen.
